schedule/yast/raid/raid1_sle_lvm.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
I think this one you have it duplicated, so it should be `lvm+raid1_sle.yaml` and you can delete the other one. Do not forget to update `bootloader_start`.

Reply:
done